spec/factory_girl/strategy/stub_spec.rb

```diff
@@ -18,6 +18,12 @@
     it { expect(subject.result(evaluation)).not_to be_new_record }
     it { expect(subject.result(evaluation)).to be_persisted }
 
+    it "clears changed attributes" do
+      evaluation.object.stubs(:clear_changes_information)
+      subject.result(evaluation)
+      expect(evaluation.object).to have_received(:clear_changes_information).once
```

Comment:
Nah, this shouldn't be necessary since there's no actual logic being tested. I'll remove it.